### ISSUE TITLE
New Trade bug 

### ISSUE THREAD

**Issue by Jon-attend** [yairm210/Unciv#2339]
Since The last update there is an issue with trades:

For instance I have two horses:
+ If I trade two horses, the display is now at 0, next turn will say: trade is no longer relevant
+ If I trade one horse, the display is now at 1, next turn the display will be at 0 and I cannot proceed to trade another one unless I save and restart in which case the display will go back to 1

all the best

**Comment by yairm210** [comment]
Huh, I'll check this out
